web/config/sync/block.block.new_weather_theme_pagetitle.yml

```diff
@@ -21,4 +21,4 @@ visibility:
   request_path:
     id: request_path
     negate: true
-    pages: '<front>'
+    pages: "<front>\r\n/point/*\r\n/node/*"
```

Comment:
We don't want the built-in page title for our previews because they just duplicate the content itself. There might be an accessibility concern here, though: without an H1 anywhere, there's no hierarchy. We could add an H1 in our templates somewhere to mitigate this, but I'm not sure where.

Reply:
We could consider an overall h1 called "Preview" at the top of all preview pages

Reply:
Yesss, smart. I'll add that to the headings overhaul PR.